src/labels/CustomLabels.labels

```diff
@@ -7296,6 +7344,22 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
         <shortDescription>Title of checklist User Onboard in Get Started</shortDescription>
         <value>Onboard Your Users</value>
     </labels>
+    <labels>
+        <fullName>gseuChecklistWelcomeNPSPDesc</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of checklist section Welcome to Nonprofit in Get Started</shortDescription>
```

Comment:
One more tiny change: Description of Welcome to Nonprofit checklist section in Get Started